Archive 2020 site, add 2021 site

geekygirlsarah opened the pull request:
This PR includes:

- Moving old images, CSS, JS, and MD to 2020 directory
- Add in 2021 files
- Include all dates I could find on planning document
- Incorporate dark purple theme
- Create new speaker photo montage background
- Double-checked all "2020" references are removed
![image](https://user-images.githubusercontent.com/2601974/109596533-72acee00-7ae4-11eb-8163-1afde07cd655.png)

geekygirlsarah commented:
I'm not sure why the Netlify build failed as I didn't change anything about it's deployment and it built file on my machine, but I can help try to debug if you want.

ertyseidohl commented on conduct.md:
```diff
@@ -1,6 +1,6 @@
+title: Code of Conduct - !!Con 2021
 ---
 
 # Code of conduct
```
I believe Josh was going to revamp the CoC for this year, make sure to check in with him before submitting.

jwise replied:
Yep, I do plan to do a rev on it.  Soon.  I promise.  I'm undecided as to whether to keep the old Code of Conduct or to wait until the new one is ready...

ertyseidohl replied:
I think we need the new CoC before the RFP goes live, since speakers should know what they're agreeing to.

geekygirlsarah replied:
The RFP isn't up right now, so the CoC could go up now as-is and could get revised before RFP. Or take the CoC down (delete the file) for this to go up, then repost when both are ready to go.

trueskawka replied:
I think it's better to iterate in smaller chunks, so I'd say let's merge this one and we can update - just having it ready before we go live with CFP.

I'm not sure what `RFP` stands for :)

ertyseidohl commented on give-a-talk.md:
```diff
@@ -1,57 +1,60 @@
 Since its launch in 2014, !!Con talks have featured everything from [glowing mushrooms](https://www.youtube.com/watch?v=T75FvUDirNM) to [game theory](https://www.youtube.com/watch?v=RHg2JIvoaq0); from [machine knitting](https://www.youtube.com/watch?v=ihqcgrR0azw) to [geolocation](https://www.youtube.com/watch?v=NvShiF4tnMM); from [live coding on an Apple II](https://www.youtube.com/watch?v=DY4t9 ...
 
-What’s an idea that delights you? Did you learn something surprising recently? Is there a tool you love that you’ve been telling everyone about? Did you do something with computers that seems impossible or amazing or just really fun? Please take this call for talk* proposals as an [invitation to meditate](https://twitter.com/akaptur/status/583115830621184000) on what you love about computing, and ...
+What’s an idea that delights you? Did you learn something surprising recently? Is there a tool you love that you’ve been telling everyone about? Did you do something with computers that seems impossible or amazing or just really fun? Please take this call for talk* proposals as an [invitation to meditate](https://twitter.com/akaptur/status/583115830621184000) on what you love about computing, and ...
```
Amazing, love it. Should we ask Allison if it's ok to link this tweet?

ertyseidohl replied:
I also think this site might go live on the 8th, so we might just end up putting the actual link here.

geekygirlsarah replied:
The tweet was already in there, so I assume since it was on in 2020, it was fine again. The change is removing the submission form link.

ertyseidohl replied:
Oh! I missed that, thanks :)

ertyseidohl commented on give-a-talk.md:
```diff
@@ -60,8 +63,10 @@ Preparing an amazing conference talk is hard work. To show our appreciation, we
 
 !!Con is committed to being an conference available to everyone. The conference hall and stage are fully wheelchair accessible. Speakers are encouraged to use their own computers to present, although we’ll also have spares on hand. We also do our best to make the conference as accessible as possible for our attendees in a variety of ways (stenography, gender-neutral restrooms, quiet spaces, etc). ...
```
We should rethink this section for a virtual con.

geekygirlsarah replied:
I can take this page down for now and when we have CFP stuff up we can repost it with re-thought links. I think getting a page up to solidify that 2021 is happening might be more important than having ALL of the info on there first.

ertyseidohl replied:
👍

ertyseidohl commented on index.md:
```diff
@@ -1,78 +1,38 @@
   [Conduct](conduct.html)
   &middot;
-  [Recordings](recordings.html)
+  [Past Recordings](recordings.html)
```
Can just remove this link, we generally add it after the con.

geekygirlsarah replied:
I figured they'd want to see past recordings perhaps, but I can remove it.

trueskawka replied:
Let's remove it :)

ertyseidohl commented on index.md:
```diff
@@ -1,78 +1,38 @@
-  &middot;
-  [Program](program.html)
-  &middot;
-  [Livestream](livestream.html)
```
Let's remove this link until the day of.

geekygirlsarah replied:
This one is not getting added in this PR (it's removed since 2020's site)

ertyseidohl replied:
🤦

ertyseidohl commented on sponsors.md:
```diff
@@ -1,92 +1,14 @@
-
-If you have other ideas, we'd love to hear about them and work with you -- feel
-free to [contact us](mailto:julia@jvns.ca)!
+We'll post more information about sponsorship levels how to contact us soon!
```
add "and"?

geekygirlsarah replied:
Fixed

ertyseidohl commented on venue-accessibility.md:
```diff
@@ -1,22 +1,22 @@
 
 ## Alcohol-free and all-ages
 
 We remain an **alcohol-free event**, as always.  All ages are welcome.
```
I think we can remove this part? Happy to leave it although I'm not sure how we would enforce it.

geekygirlsarah replied:
I'll update it.

geekygirlsarah commented:
Rather than have old pages with all the content removed and we can add links to the home page later, I just went ahead and deleted the files (as they're easy to restore from Git). We can bring them back when we're ready with the info to publish.

Only remaining pages are index.md, conduct.md (which can get revised when Josh has it), and recordings.md (for past recordings, which we can take down this page when we post all the other info on the site)

lkuper commented:
@geekygirlsarah Looking good!! 💜 The "157 past speakers" number at the bottom ought to be updated to whatever the new total is (157 + however many there were at !!Con 2020).

geekygirlsarah commented:
> @geekygirlsarah Looking good!! 💜 The "157 past speakers" number at the bottom ought to be updated to whatever the new total is (157 + however many there were at !!Con 2020).

I fixed this. (Actually randomly thought about it this morning and fixed it before I saw the comment!)

trueskawka commented on recordings.md:
```diff
@@ -1,30 +1,8 @@
 ---
 
-## Recordings
+## Past Recordings
```
If we remove the link on main page, will folks be able to navigate here?

geekygirlsarah replied:
Nope.

trueskawka reviewed (approved):
It looks good! I had a couple of minor comments, but I'd prefer to iterate on a smaller PR than block the change that has most of what we need :)

geekygirlsarah commented:
>I think it's better to iterate in smaller chunks, so I'd say let's merge this one and we can update - just having it ready before we go live with CFP.
>
>I'm not sure what RFP stands for :)

RFP = request for proposals, I assume.

And yes, getting this up and iterating over small changes is what I think. Probably good to officially get the word out on 2021 now is good even if there's not much info up there yet. It can start people thinking about sponsorships, thinking of talks to submit, etc.

geekygirlsarah commented:
Psst... someone want to merge this so there's a 2021 site...  😁

jwise commented:
I sent you an organization invite! I'm on my phone so I do not have slack access but you should be able to merge it yourself now!
